docs/web/release-notes-5.0.md

```diff
@@ -328,4 +328,8 @@ For more details, see <https://github.com/thelfer/tfel/issues/476>
 
 ## Issue 370: [tfel-utilities] Support for `C++` digit separator
 
-For more details, see <https://github.com/thelfer/tfel/issues/370>
\ No newline at end of file
+For more details, see <https://github.com/thelfer/tfel/issues/370>
+
+## Issue 632: [tfel-material] add-computations-of-the-eshelby-tensors 
+
+For more details, see <https://github.com/thelfer/tfel/issues/632>
```

Comment:
This is ok for solving small issues, not new features for which we create at dedicated section.

Would you create a section named  `New TFEL/Material feature` similar to the already existing `New TFEL/Math feature` and populate it a bit.

See with Renaud if we shall add an akwowledgment to the ANR.